src/flint/fmpz.jl

```diff
@@ -1275,8 +1275,10 @@ Return a tuple $g, s, t$ such that $g$ is the greatest common divisor of $a$
 and $b$ and integers $s$ and $t$ such that $g = as + bt$.
 """
 function gcdx(a::ZZRingElem, b::ZZRingElem)
-  # Just to conform with Julia's definition
-  a == b == 0 && return zero(ZZ), one(ZZ), zero(ZZ)
+  if VERSION < v"1.12.0"
```

Comment:
Actually ideally we'd like to have this change now, and in a way that fixes our CI tests against Julia nightly. I think that means using something like
```suggestion
  if VERSION < v"1.12.0-DEV.420"
```
but @lgoettgens may know better (I got the version from the CI logs, so this may not be the exact version they made the change)

Reply:
Sounds good. We should perhaps also insert `@static` before the if-statement, no?

Reply:
Yes, that looks right. Thanks!